packages/instantsearch.js/src/components/SearchBox/SearchBox.tsx

```diff
@@ -184,6 +189,9 @@ class SearchBox extends Component<
             spellCheck="false"
             maxLength={512}
             onInput={this.onInput}
+            // see: https://github.com/preactjs/preact/issues/1978
+            // eslint-disable-next-line react/no-unknown-property
+            oncompositionend={this.onInput}
```

Comment:
not relevant now, but I wonder how we'll deal with these differences with only one JSX source